package.json

```diff
@@ -1,32 +1,24 @@
 {
   "name": "to-do-list",
   "version": "1.0.0",
-  "description": "simple to-do list app ",
-  "main": "server.js",
-  "directories": {
-    "doc": "docs"
-  },
+  "description": "simple to-do list app",
+  "main": "app/server.js",
   "scripts": {
-    "test": "mocha && mocha test/features",
     "start": "node app/server.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Jojograndjojo/To-do-List.git"
```

Comment:
Think we should change to this?  Same for all occurrences

https://github.com/vannio/unicorn-framework